source/gamemodes/BoostMod.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include "../RocketPlugin.h"
+
+
+class BoostMod : public RocketGameMode
+{
+private:
+	float boostConversionPercentage = 100.0f;
+
+	void onTick(ServerWrapper server, void* params);
+	void OnBoostPickUp(ActorWrapper caller, void* params, std::string funcName);
+	void setMaxBoost(int newBoost);
+	std::vector<TeamWrapper> getTeams();
```

Comment:
If you want to make a helper function that gets the current teams for you, it would better to place it in `
RocketPlugin.cpp` so every game mode can potentially use it.